yt/testing.py

```diff
@@ -1269,3 +1269,47 @@ def compare_dobj_selection(self, dobj):
                 obj_results.append(chunk[ptype, "particle_position"])
             obj_results = np.concatenate(obj_results, axis = 0)
             assert_equal(sel_pos, obj_results)
+
+    def run_defaults(self):
+        """
+        This runs lots of samples that touch different types of wraparounds.
```

Comment:
Can you either add inline comments or to this docstring the specific comparisons that are being made. That way a future frontend writer will know what selectors they need to have coverage for if they want to use non-default comparisons in their testing.

Reply:
Sure.